datasets/coco2014/metadata.json

```diff
@@ -297,13 +297,11 @@
         },
         {
           "name": "bbox",
-          "@type": "ml:Field",
-          "description": "The bounding box around annotated object[s].",
+          "@type": "ml:BoundingBox",
+          "format": "XYWH",
```

Comment:
I think format should be inside the source, like it was before.

Reply:
done

Reply:
done